src/app/components/Buttons/Buttons.types.ts

```diff
@@ -0,0 +1,10 @@
+export type PrimaryButtonProps = {
+  buttonLabel: string;
+};
+
+export type IconButtonProps = {
+  classNames?: string;
+  iconPath: string;
+  alt: string;
+  onClick?: (() => void) | undefined;
```

Comment:
void itself means that nothing is returned from the func. so 
`onClick?: () => void;` is valid option

Reply:
Ahh ok I see
I need to tinker around with the linter's config